auth/src/main/java/com/firebase/ui/auth/ErrorCodes.java

```diff
@@ -0,0 +1,20 @@
+package com.firebase.ui.auth;
+
+/**
+ * Error codes retrieved from {@link IdpResponse#getErrorCode()}.
+ */
+public final class ErrorCodes {
+    private ErrorCodes() {
+        // We don't want people to initialize this class
+    }
+
+    /**
+     * Sign in failed due to lack of network connection
+     **/
+    public static final int NO_NETWORK = 10;
```

Comment:
Isn't this one actually a result code and not an error?  We still need to return this via onActivityResult in order to honor our previous API contract.